chef/cookbooks/ceph/recipes/radosgw_ha.rb

```diff
@@ -31,18 +31,30 @@
 # Avoid races when creating pacemaker resources
 crowbar_pacemaker_sync_mark "wait-ceph-radosgw_ha_resources"
 
-service_name = "ceph-radosgw"
+transaction_objects = []
 
+service_name = "ceph-radosgw"
 pacemaker_primitive service_name do
   agent node[:ceph][:ha][:radosgw][:agent]
   op node[:ceph][:ha][:radosgw][:op]
-  action :create
+  action :update
   only_if { CrowbarPacemakerHelper.is_cluster_founder?(node) }
 end
+transaction_objects << "pacemaker_primitive[#{service_name}]"
 
-pacemaker_clone "cl-#{service_name}" do
+clone_name = "cl-#{service_name}"
+pacemaker_clone clone_name do
   rsc service_name
-  action [:create, :start]
+  meta ({ "clone-max" => CrowbarPacemakerHelper.num_corosync_nodes(node) })
```

Comment:
I think it's a bug in rubocop. At least, I can't find a way to make that work (removing `()` doesn't work, removing `{}` doesn't work, etc.)